### PR TITLE
Update tor_dnsresolver_blacklist

### PR DESCRIPTION
**Pull request by goll**
Updates the blacklisted DNS resolvers with changes and IPv6 additions.

### CODE REVIEW

**Comment by nusenu** [comment]
hello,
thanks for your interest, I'll comment directly next to the changes.

Did the currently shipped GPG key cause any issues for you?

File: defaults/main.yml [hunk 1]
```diff
@@ -130,9 +137,9 @@ tor_exit_notice_file: tor-exit-notice.html
 #
 # Quad9 DNS IPs:
 #  9.9.9.9
-#  9.9.9.10
-#  149.112.112.10
```
Comment (nusenu) [inline defaults/main.yml]: Can you elaborate on the reasoning behind this removal?
These IPs quad9 resolvers as per: https://www.quad9.com/faq/

File: defaults/main.yml [hunk 2]
```diff
@@ -113,13 +113,20 @@ tor_exit_notice_file: tor-exit-notice.html
...
+# Google DNS IPs:
+#  8.8.8.8
+#  8.8.4.4
+#  2001:4860:4860::8888
+#  2001:4860:4860::8844
+#  taken from: https://developers.google.com/speed/public-dns/docs/using#google_public_dns_ip_addresses
 #
 # OpenDNS IPs:
-#  208.67.222.123
-#  208.67.220.123
```
Comment (nusenu) [inline defaults/main.yml]: Did you verify whether the former IPs no longer work?

File: tasks/configure.yml [hunk 3]
```diff
@@ -54,9 +55,8 @@
    - "{{ tor_ports }}"
 
 - name: Abort if Ed25519 master keys are on the relay
-  fail: msg="
-
-            Ed25519 MASTER KEY detected on the relay - it is NOT supposed to be there! Aborting."
+  fail:
+    msg: "Ed25519 MASTER KEY detected on the relay - it is NOT supposed to be there! Aborting."
```
Comment (nusenu) [inline tasks/configure.yml]: I added these newlines back then to have the message be better visible from the rest, do the newlines no longer work?

File: defaults/main.yml (hunk 1 above)
Comment (goll) [reply defaults/main.yml]: thank you for the correct url, I just followed the existing link to confirm IPs, fixed

File: defaults/main.yml (hunk 2 above)
Comment (goll) [reply defaults/main.yml]: good point, they do still resolve, reverting removal

File: tasks/configure.yml (hunk 3 above)
Comment (goll) [reply tasks/configure.yml]: yeah, sorry about that, just assumed it was an accidental newline, reverted the format

**Comment by goll** [comment]
regarding the key, it was mostly syncing with the upstream repository as the current file doesn't exist anymore: https://deb.torproject.org/torproject.org/
```
Then add the gpg key used to sign the packages by running the following commands at your command prompt:

    # curl https://deb.torproject.org/torproject.org/A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89.asc | gpg --import
    # gpg --export A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89 | apt-key add -
```
since relayor doesn't yet handle those keyring packages the key should always reflect upstream afaik:
```
We provide a Debian package to help you keep our signing key current. It is recommended you use it. Install it with the following commands:

    # apt update
    # apt install tor deb.torproject.org-keyring
```

**Comment by nusenu** [comment]
> since relayor doesn't yet handle those keyring packages 

the keyring package is supposed to be installed by relayor, does that not work for you?

**Comment by goll** [comment]
oops, ignore that keyring comment (local issue), you're right regarding the keyring package

**Comment by nusenu** [comment]
please make this PR only about the DNS blacklist changes and keep the other changes separate from it, thank you!

**Comment by goll** [comment]
thank you and sorry for the noise, hope the commit looks good now

**Comment by nusenu** [comment]
thanks for making this just about the dns blacklist

could you add:
2620:fe::10 – 2620:fe:0:0:0:0:0:10
2620:fe::fe:10 – 2620:fe:0:0:0:0:fe:10
from
https://www.quad9.com/faq/
as well?
thanks!

**Comment by goll** [comment]
oops, missed the secondary IPv6 addresses the first time I looked, thanks, updated!

**Comment by nusenu** [comment]
the regex will match much more then we blacklisted before but it will do for now, merged.

thank you for your contribution

**Pull request closed without merging**